pkg/build/build.go

```diff
@@ -37,13 +38,17 @@ func Build(manifest model.Manifest, githubToken string) error {
 		}
 	}
 
-	if err := SanitizeAllCharts(manifest); err != nil {
```

Comment:
I think we want this all of the time. We still want to local helm chart files sanitized, just skip the step to package them with `helm package`.

So just HelmCharts in the 'if'

Reply:
done